lib/devise/models/database_authenticatable.rb

```diff
@@ -37,11 +37,12 @@ def self.required_fields(klass)
       # the hashed password.
       def password=(new_password)
         @password = new_password
-        self.encrypted_password = password_digest(@password) if @password.present?
+        self.encrypted_password = password_digest(@password)
```

Comment:
@lucasmazza Yes, I know that. Since we have put `NOT NULL constraint` for `encrypted_password` on migration. So, I am generating password digest  `nil`  values as well.